commands/commands.go

```diff
@@ -334,6 +334,18 @@ func isCommandEnabled(cfg *config.Configuration, cmd string) bool {
 	return cfg.Os.Bool(fmt.Sprintf("GITLFS%sENABLED", strings.ToUpper(cmd)), false)
 }
 
+func requireGitVersion() {
+	minimumGit := "1.8.2"
```

Comment:
I wonder if it would be worth elevating this to a method argument, and having a const in the `commands` package.

This would allow us to easily require different versions of Git for different commands, but I'm not sure that's behavior we need.